delta-api/src/main/java/io/cdap/delta/api/Sequenced.java

```diff
@@ -26,20 +26,28 @@
 public class Sequenced<T> {
   private final T event;
   private final long sequenceNumber;
+  private static final long UNSUPPORTED_SEQUENCE = -1;
 
+  public Sequenced(T event) {
```

Comment:
javadoc would be good indicating when to use this constructor vs next.

Reply:
done :)